src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -1338,6 +1281,94 @@ await AddDependencyFlowEventsAsync(
             throw;
         }
     }
+    private async Task<CodeFlowResult?> ExecuteCodeFlowAsync(
+        InProgressPullRequest? pr,
+        SubscriptionUpdateWorkItem update,
+        SubscriptionDTO subscription,
+        BuildDTO build,
+        string prHeadBranch,
+        bool forceUpdate,
+        bool isForwardFlow)
+    {
+        _logger.LogInformation(
+            "{direction}-flowing build {buildId} for subscription {subscriptionId} targeting {repo} / {targetBranch} to new branch {newBranch}",
+            isForwardFlow ? "Forward" : "Back",
+            build.Id,
+            subscription.Id,
+            subscription.TargetRepository,
+            subscription.TargetBranch,
+            prHeadBranch);
```

Comment:
Unrelated to this PR but a piece of information I really need when reading the logs:

```suggestion
        _logger.LogInformation(
            "{direction}-flowing build {buildId} of {sourceRepo} for subscription {subscriptionId} targeting {targetRepo} / {targetBranch} to new branch {newBranch}",
            isForwardFlow ? "Forward" : "Back",
            build.Id,
            subscription.SourceRepository,
            subscription.Id,
            subscription.TargetRepository,
            subscription.TargetBranch,
            prHeadBranch);
```